src/Components/Onboarding/ItemLink.tsx

```diff
@@ -19,7 +19,7 @@ const Link = styled.a`
 `
 
 const Avatar = styled.img`
-  padding: 10px 15px 10px 10px;
+  margin: 10px 15px 10px 10px;
```

Comment:
We want to use `margin` instead of `padding` to prevent the images from being distorted in force.